sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1649,8 +1658,20 @@ def fit(self, X, y, sample_weight=None):
                 for l1_ratio in l1_ratios
             ]
         else:
+            # Making sure alphas entries are scalars.
+            if np.isscalar(alphas):
+                alphas = check_scalar_alpha(alphas, "alphas")
```

Comment:
I do not think we should set with `check_scalars`:

```suggestion
                check_scalar_alpha(alphas, "alphas")
```

It gives the impression that `check_scalar_alpha` does something with `alphas`, but all it does is return the original `alphas`.